commit a9c9593126c7
Author: Michael Froman <mfroman@mozilla.com>
Date:   Wed Feb 14 14:24:50 2018 -0600

    Bug 1436759 - Release NrIceMediaStreams on sts thread in transport_unittests.cpp. r=bwc
    
    MozReview-Commit-ID: BLasqfQJxw
    
    --HG--
    extra : rebase_source : bca09672917ce5fc4a5450864c9f461952847b3f
---
 media/mtransport/test/transport_unittests.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git media/mtransport/test/transport_unittests.cpp media/mtransport/test/transport_unittests.cpp
index 167a14be1fd52..4bf6195122182 100644
--- media/mtransport/test/transport_unittests.cpp
+++ media/mtransport/test/transport_unittests.cpp
@@ -481,6 +481,7 @@ class TransportTestPeer : public sigslot::has_slots<> {
       flow_ = nullptr;
     }
     ice_ctx_ = nullptr;
+    streams_.clear();
   }
 
   void DisconnectDestroyFlow() {
